The declaration of basename below conflicts with the one in <string.h>
in newer gnu libc over the use of const.

diff -Naurb version-binutils-2.17.orig/binutils-2.17/include/libiberty.h version-binutils-2.17/binutils-2.17/include/libiberty.h
--- version-binutils-2.17.orig/binutils-2.17/include/libiberty.h	2009-08-20 13:52:49.000000000 -0500
+++ version-binutils-2.17/binutils-2.17/include/libiberty.h	2010-06-15 23:43:37.492828432 -0500
@@ -98,7 +98,9 @@
    is 1, we found it so don't provide any declaration at all.  */
 #if !HAVE_DECL_BASENAME
 #if defined (__GNU_LIBRARY__ ) || defined (__linux__) || defined (__FreeBSD__) || defined (__OpenBSD__) || defined(__NetBSD__) || defined (__CYGWIN__) || defined (__CYGWIN32__) || defined (__MINGW32__) || defined (HAVE_DECL_BASENAME)
+#if 0
 extern char *basename (const char *);
+#endif
 #else
 /* Do not allow basename to be used if there is no prototype seen.  We
    either need to use the above prototype or have one from
